samples/WebsocketConnect/README.md

```diff
@@ -48,3 +48,139 @@ If you wish to use the latest SDK release to run the sample rather than using th
 ```sh
 mvn -P latest-release compile exec:java -pl samples/WebsocketConnect -Dexec.mainClass=websocketconnect.WebsocketConnect -Dexec.args="--endpoint <endpoint> --signing_region <signing region>"
 ```
+
+## Alternate connection configuration methods supported by AWS IoT Core
+
+### MQTT over WebSockets with static AWS credentials
+
+With a help of a static credentials provider your application can use a fixed set of AWS credentials. For that, you need
+to instantiate the `StaticCredentialsProviderBuilder` class and provide it with the AWS credentials. The following code
+snippet demonstrates how to set up an MQTT3 connection using static AWS credentials for SigV4-based authentication.
+
```

Comment:
Trivial: Should we simplify this snippet? Include only what's absolutely necessary to establish this connection and remove references to "cmdData". 

e.g. remove withCertificateAuthorityFromPath, withCleanSession, withProtocolOperationTimeouts, (does this need client ID?). Change all cmdData related things to <clientID> or <endpoint>.

Reply:
Regarding `cmdData`: the idea was to provide a code snippet that could be injected into the sample as is. But considering your another comment that we also need to parse additional options, I tend to agree with you on this.